app/.nvmrc

```diff
@@ -1 +0,0 @@
-v13.10.1
```

Comment:
odd that this doesn't break the actions workflow? https://github.com/avimoondra/fishbowl/blob/master/.github/workflows/lint.yml#L13

referencing .nvmrc, which now moved to top level dir?

Reply:
_Oh,_ good catch. This actually isn't working as expected, i.e.

```
cat: .nvmrc: No such file or directory
```

I'll fix it in another PR.